tests/integration/api_tests.py

```diff
@@ -154,5 +159,11 @@ def test_mfa_verify(self):
         response = self.auth_client.create_verify_two_factor(ACCOUNT_ID, body)
         self.assertTrue(isinstance(response.body.valid, bool))
 
+    def test_tn_lookup(self):
+        body = AccountsTnlookupRequest()
+        body.tns = [PHONE_NUMBER_OUTBOUND]
+        response = self.tn_lookup_client.create_tn_lookup_request(ACCOUNT_ID, body)
+        self.assertTrue(response.status_code == 202)
```

Comment:
Can you add something for `get_tn_lookup_result` too? It can be within this function too, similar to the get call flow

Reply:
Added